contracts/ExitHandler.sol

```diff
@@ -225,6 +234,19 @@ contract ExitHandler is IExitHandler, DepositHandler {
 
       currentExit = exits[utxoId];
 
+      // exits of deleted periods are not payed out but simply deleted
+      if (exitRoots[utxoId] > 0) {
+        uint32 timestamp;
+        (, timestamp,,) = bridge.periods(exitRoots[utxoId]);
+        if (timestamp == 0) {
+          // stake goes to governance contract
+          address(uint160(bridge.admin())).send(currentExit.stake);
```

Comment:
why so? Wouldn't it be locked there forever?

Reply:
ah Yeah, needs an update to governance contract

Reply:
@troggy  done